RadialGM.pro

```diff
@@ -178,4 +178,5 @@ FORMS += \
 RESOURCES += \
     images.qrc
 
-DISTFILES +=
+DISTFILES += \
+  Submodules/enigma-dev/events.ey
```

Comment:
I don't know if this was necessary, the documentation says it only works on Unix, and you've already put it in the resources file.
https://stackoverflow.com/a/38102991

Reply:
qtcreator did this not me

Reply:
Hrm idk then.